tests/metrics/estimate_gas.py

```diff
@@ -153,7 +153,12 @@ def estimate_gas(analyzer: AnalyzeGas = None) -> None:
     log = Logger(AnalyzeGas.LOG_NAME)
 
     # Blockchain
-    testerchain, agents = TesterBlockchain.bootstrap_network()
+    economics = StandardTokenEconomics()
+    economics.base_penalty = MIN_ALLOWED_LOCKED - 1
```

Comment:
I'd love to see these values either passed into the economics `__init__` or hard-coded onto their own class altogether.

Reply:
✔️